ffxivbot/pika_rabbit.py

```diff
@@ -160,7 +160,7 @@ def req_url(params):
                 message = params["message"]
                 message = re.sub(r"\[CQ:at,qq=(.*?)\]", "<@\g<1>>", message)
                 print("message 1 >>> {}".format(message))
-                img_pattern = r"\[CQ:image,(?:cache=.,)?file=(.*?)\]"
+                img_pattern = r"\[CQ:image,(?:cache=.,)?file=(.*?)(?:\]|,.*?\])"

```

Comment:
What will the tailing `.*?` capture?

Reply:
Capture other parameters，such as `destruct=1`